### PR TITLE
Add an option for deleting a file in the localStorage

### PR DESCRIPTION
**Pull request by CedricJean**

### CODE REVIEW

**Comment by CedricJean** [comment]
capture

File: src/components/HeaderDrawer/index.js [hunk 1]
```diff
@@ -21,21 +22,28 @@ const useStyles = makeStyles({})
...
+  const c = useStyles();
   const onDrop = useCallback((acceptedFiles) => {
     onOpenFile(acceptedFiles[0])
   }, [])
 
-  const { getRootProps, getInputProps } = useDropzone({ onDrop })
+  function onDeleteFile(i) {
+    console.log(i);
+    changeRecentItems( recentItems.filter(function(oneRecentFile){console.log(oneRecentFile.id);return oneRecentFile.id!==i}));
+    localStorage.removeItem('recentItems');
```
Comment (seveibar) [inline src/components/HeaderDrawer/index.js:41]: you don't need to remove prior to the `setItem`

File: src/components/HeaderDrawer/index.js [hunk 2]
```diff
@@ -21,21 +22,28 @@ const useStyles = makeStyles({})
...
-
+  const c = useStyles();
   const onDrop = useCallback((acceptedFiles) => {
     onOpenFile(acceptedFiles[0])
   }, [])
 
-  const { getRootProps, getInputProps } = useDropzone({ onDrop })
+  function onDeleteFile(i) {
+    console.log(i);
+    changeRecentItems( recentItems.filter(function(oneRecentFile){console.log(oneRecentFile.id);return oneRecentFile.id!==i}));
```
Comment (seveibar) [inline src/components/HeaderDrawer/index.js:40]: prefer the arrow syntax for inline functions, e.g.

```javascript
changeRecentItems( recentItems.filter(oneRecentFile => oneRecentFile.id!==i))
```

File: src/components/HeaderDrawer/index.js [hunk 3]
```diff
@@ -21,21 +22,28 @@ const useStyles = makeStyles({})
...
+    localStorage.removeItem('recentItems');
+    localStorage.setItem('recentItems',JSON.stringify(recentItems));
```
Comment (seveibar) [inline src/components/HeaderDrawer/index.js:42]: this is implicitly handled in the `useLocalStorage` method, so you shouldn't need to interact directly with `localStorage` (no setItem/deleteItem required :)

File: src/components/HeaderDrawer/index.js [hunk 4]
```diff
@@ -21,21 +22,28 @@ const useStyles = makeStyles({})
...
-  const c = useStyles()
-
+  const c = useStyles();
   const onDrop = useCallback((acceptedFiles) => {
     onOpenFile(acceptedFiles[0])
   }, [])
 
-  const { getRootProps, getInputProps } = useDropzone({ onDrop })
+  function onDeleteFile(i) {
+    console.log(i);
```
Comment (seveibar) [inline src/components/HeaderDrawer/index.js:39]: :x: :wink:

Comment (seveibar) [reply src/components/HeaderDrawer/index.js:39]: ^ the cryptic comment above means we probably don't need the `console.log` haha

File: src/components/HeaderDrawer/index.js (hunk 1 above)
Comment (CedricJean) [reply src/components/HeaderDrawer/index.js:41]: Ok i will change this in this case. Sry for the beginner mistake im starting

**Comment by CedricJean** [comment]
The change is made

**Comment by seveibar** [comment]
Hey @CedricJean, this looks good (somehow a bunch of formatting things got merged into this but that's okay it must be because me or @puskuruk pushed something without letting the tests pass :man_shrugging: )

Thanks for the contribution!! :1st_place_medal:

**Pull request merged**